core/lib/sqlcipher/CMakeLists.txt

```diff
@@ -1,7 +1,16 @@
 project(sqlcipher)
 
+message(STATUS "Configuring SQLCipher:")
+
 add_library(sqlcipher STATIC sqlite3.c sqlite3.h sqlite3ext.h)
 
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Android")
+  # On Android, temporary directories are a bit special and are not located in the same place, such
+  # as /tmp or /usr/tmp; for this reason, we ask temporary files to be stored in memory instead
+  message("Compiling SQLCipher for Android without temporary directory disabled (all-on-memory).")
+  add_definitions(-DSQLITE_TEMP_STORE=3)
```

Comment:
Better place for this line is core/src/CMakeLists.txt, and we already have ANDROID branch there.
It would be better to not touch this file as it is kind on 3rd-party

Reply:
Yeah, you are totally right. I’ll change that.

Reply:
`core/lib/CMakeLists.txt`, you meant.